packages/migration/src/mods/7.0.0/comboboxPropsChange.ts

```diff
@@ -0,0 +1,16 @@
+import { IMigrationOptions, migration, ModResult } from '../../migration';
+import { mod } from 'riceburn';
+import { renameJsxProp } from 'riceburn/lib/mods/renameJsxProp';
+
+export default migration(
+  'remove use of deprecated ComboBox props',
+  (opts: IMigrationOptions): ModResult[] => {
+    return mod('**/*.tsx', opts)
+      .asTypescript((node, modder) => {
+        return renameJsxProp('ComboBox', 'value', 'text')(node, modder);
+      })
+      .asTypescript((node, modder) => {
+        return renameJsxProp('ComboBox', 'onChanged', 'onChange')(node, modder);
```

Comment:
This isn't a simple rename, the order of callback parameters also changes.
```
  onChange?: (event: React.FormEvent<IComboBox>, option?: IComboBoxOption, index?: number, value?: string) => void;

  onChanged?: (option?: IComboBoxOption, index?: number, value?: string, submitPendingValueEvent?: any) => void;
```

Reply:
ah good catch; thanks!